src/components/DateInput/index.test.js

```diff
@@ -23,26 +23,26 @@ describe('DateInput component', () => {
 
   it('calls onInputChange when the input value changes', async () => {
     expect(inputChangeFn).not.toHaveBeenCalled();
-    await userEvent.type(screen.getByRole('textbox'), '4')
+    await userEvent.type(screen.getByRole('textbox'), '4');
```

Comment:
Absolutely no changes required here, but just to add a note: 

prettier githook currently does not operate on our tests but this will be something I fix once the dust has settled on v10...in case anyone is wondering why these semicolons were not here in the first place, and why they were added (probably an editor plugin)

Reply:
I actually added those semicolons myself, haha. 😅